cli/delete.go

```diff
@@ -77,6 +78,15 @@ func delete(containerID string, force bool) error {
 			return fmt.Errorf("Container still running, should be stopped")
 		}
 
+		// Force killing the container in case the container is still
+		// running, but don't throw an error if it failed for any
+		// reason because that's the behavior of "--force". Also, a
+		// previous call to kill might have been performed earlier,
+		// causing this one to fail.
+		if err := vci.KillContainer(sandboxID, containerID, syscall.SIGKILL, true); err != nil {
```

Comment:
deleteContainer(forceStop==true) will also send SIGKILL to the container. Is there any reason that we need to do it here before virtcontainers does it?

Reply:
well, deleteSandbox() also sends it. I do not see why we need it here.

Reply:
Where are we killing the containers ?

Reply:
Oh nevermind, I forgot this was handled through virtcontainers...

Reply:
I have addressed your comment.